### PR TITLE
EOS-16540: add report for  pacemaker 1.1 vs 2.0 changelog

### PR DESCRIPTION
**Pull request by azheregelya**
- Analyzed ClusterLabs changelog documents
- Write down changes that affect LR1, LR2 releases
- Explain which activities are affected by aforementioned changes

Signed-off-by: Andrei Zheregelia <andrei.zheregelia@seagate.com>

### CODE REVIEW

**Comment by azheregelya** [comment]
Hi @mukul-seagate11 any reason to remove reviewers?

**Comment by mukul-seagate11** [comment]
> Hi @mukul-seagate11 any reason to remove reviewers?

not sure how the reviewers are removed as I am not aware so let me add again

**Comment by azheregelya** [comment]
@mukul-seagate11 , Thank you  :)

File: Pacemaker/Pacemaker-2-changelog.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+- [Differences between Pacemaker 1.1 and 2.0](#org31d2d10)
+  - [Generic points:](#orgbd42bfc)
+    - [2.0 breaks rolling upgrades from Pacemaker 1.1.10 and earlier on corosync 2 and greater (and of course from any version on other stacks)](#orga3109e6)
+    - [Only 2.0 will be actively developed from this point, but at least some bug fixes will be backported to the 1.1 branch for the foreseeable future, and we will eventually do at least one more 1.1 release with those fixes](#org324eeee)
```
Comment (ajaysrivas) [inline Pacemaker/Pacemaker-2-changelog.md:4]: Any idea about end of life date for pacemaker 1.1?

**Review by indrajitzagade — APPROVED** [review]

File: Pacemaker/Pacemaker-2-changelog.md (hunk 1 above)
Comment (azheregelya) [reply Pacemaker/Pacemaker-2-changelog.md:4]: I didn't manage to find any information or official statement about that.
Asked a question in user mail group. Waiting for reply.

Comment (azheregelya) [reply Pacemaker/Pacemaker-2-changelog.md:4]: According to reply I received the Pacemaker 1.1 support is already over.
Details: https://lists.clusterlabs.org/pipermail/users/2021-February/028423.html

**Pull request merged**